zebra-grpc/Cargo.toml

```diff
@@ -14,8 +14,19 @@ keywords = ["zebra", "zcash"]
 # Must be one of <https://crates.io/category_slugs>
 categories = ["cryptography::cryptocurrencies"]
 
+[[bin]] # Bin to run the Scanner gRPC server
+name = "scanner-grpc-server"
+path = "src/server.rs"
+
 [dependencies]
 
 tonic = "0.10.2"
+prost = "0.12.3"
+tokio = { version = "1.35.1", features = ["macros", "rt-multi-thread"] }
 
 zebra-scan = { path = "../zebra-scan", version = "0.1.0-alpha.1" }
```

Comment:
I think we should reverse this dependency to use zebra-scan as the entry point for the scanner process.

Reply:
Can you explain the reasons of this ?

Reply:
To avoid closely coupling zebra-scan/zebra-grpc so that zebra-scan is more extensible, and the reasons in https://github.com/ZcashFoundation/zebra/pull/8178#discussion_r1462346185